examples/torch/pearl_ml1_push.py

```diff
@@ -0,0 +1,135 @@
+"""PEARL ML1 example."""
+
+import os
+
+from metaworld.benchmarks import ML1
+
+from garage.envs import GarageEnv, normalize
+from garage.experiment import LocalRunner, SnapshotConfig, wrap_experiment
+from garage.experiment.deterministic import set_seed
+from garage.experiment.meta_evaluator import MetaEvaluator
+from garage.experiment.task_sampler import SetTaskSampler
+from garage.sampler import LocalSampler
+from garage.torch.algos import PEARL
+from garage.torch.algos.pearl import PEARLWorker
+from garage.torch.embeddings import MLPEncoder
+from garage.torch.policies import (ContextConditionedPolicy,
+                                   TanhGaussianMLPPolicy)
+from garage.torch.q_functions import ContinuousMLPQFunction
+import garage.torch.utils as tu
+
+algo_params = dict(
+    num_epochs=1000,
+    num_train_tasks=50,
+    num_test_tasks=10,
+    latent_size=7,
+    encoder_hidden_sizes=[200, 200, 200],
+    net_size=300,
+    meta_batch_size=16,
+    num_steps_per_epoch=4000,
+    num_initial_steps=4000,
+    num_tasks_sample=15,
+    num_steps_prior=750,
+    num_extra_rl_steps_posterior=750,
+    num_steps_per_eval=450,
+    batch_size=256,
+    embedding_batch_size=64,
+    embedding_mini_batch_size=64,
+    max_path_length=150,
+    reward_scale=10.,
+    use_information_bottleneck=True,
+    use_next_obs_in_context=False,
+    use_gpu=True,
+)
+
+
+# pylint: disable=unused-argument
+@wrap_experiment
+def torch_pearl_ml1_push(ctxt=None, seed=1, **params):
+    """Train PEARL with ML1 environments.
+
+    Args:
+        ctxt (garage.experiment.ExperimentContext): The experiment
+            configuration used by LocalRunner to create the snapshotter.
+        seed (int): Used to seed the random number generator to produce
+            determinism.
+        params (dict): Parameters for PEARL.
+
+    """
+    set_seed(seed)
+    # create multi-task environment and sample tasks
+    env_sampler = SetTaskSampler(lambda: GarageEnv(
+        normalize(ML1.get_train_tasks('push-v1'))))
+    env = env_sampler.sample(params['num_train_tasks'])
+
+    test_env_sampler = SetTaskSampler(lambda: GarageEnv(
+        normalize(ML1.get_test_tasks('push-v1'))))
+    snapshot_config = SnapshotConfig(snapshot_dir=os.path.join(
```

Comment:
is there a reason that you can't use the default here? AFAICT this is the default SnapshotConfig

Reply:
You should just be able to pass `ctxt` instead of `snapshot_config`. They implement the same fields.